plugins/meta/aos-firewall/firewall/firewall.go

```diff
@@ -403,14 +499,43 @@ func (f *Firewall) formatIptablesRequest(chain *AccessChain) (chainFilters []ipt
 						src:      dchain.Address.IP.String(),
 						protocol: rule.Protocol,
 						dPorts:   rule.Ports,
-						jump:     "ACCEPT"})
+						jump:     "ACCEPT",
+					})
+
+					// Allow traffic forward between containers on different subnets
+					outputParams = append(outputParams, iptablesRequest{
```

Comment:
We need it only if there are allowed connections to the foreign subnet. Don't we? If so, can we optimize this functionality in order to allow foreign subnet access only if there are allowed connections?

Reply:
in added only when we use allow port connection from container